Release SDK32

tsapeta opened the pull request.

dsokal committed:
[tools-public] fix generate-dynamic-macros-android.sh

tsapeta committed:
[sdk] bump react-native-reanimated JS version to 1.0.0-alpha.11

tsapeta committed:
[ncl] add missing setMinimumIntervalAsync call to background fetch example

sjchmiela committed:
[packages] Do not publish babel.config.js in universal modules (#3099)

# Why

With `babel.config.js` published inside the packages, Jest transpiles their source code with included configuration (which isn't, and isn't supposed to be, valid outside of Expo monorepo).

# How

Added `.npmignore` file, excluding `babel.config.js` to `expo-module-scripts/templates`, ran:

```sh
# in packages
for package in ./*; do pushd $package; yarn; popd; done
```

# Test Plan

- [x] Publish prerelease packages
- [x] Confirm that tests work now out of the box

sjchmiela committed:
Publish SDK32

expo-ads-admob@2.0.0
expo-app-auth@2.0.0
expo-app-loader-provider@1.0.0
expo-background-fetch@1.0.0
expo-barcode-scanner@2.0.0
expo-barcode-scanner-interface@2.0.0
expo-camera@2.0.0
expo-camera-interface@2.0.0
expo-constants@2.0.0
expo-constants-interface@2.0.0
expo-contacts@2.0.0
expo-core@2.0.0
expo-errors@1.0.0
expo-face-detector@2.0.0
expo-face-detector-interface@2.0.0
expo-file-system@2.0.0
expo-file-system-interface@2.0.0
expo-firebase-analytics@2.0.0
expo-firebase-app@2.0.0
expo-firebase-auth@2.0.0
expo-firebase-crashlytics@2.0.0
expo-firebase-database@2.0.0
expo-firebase-firestore@2.0.0
expo-firebase-functions@2.0.0
expo-firebase-instance-id@2.0.0
expo-firebase-invites@2.0.0
expo-firebase-links@2.0.0
expo-firebase-messaging@2.0.0
expo-firebase-notifications@2.0.0
expo-firebase-performance@2.0.0
expo-firebase-remote-config@2.0.0
expo-firebase-storage@2.0.0
expo-font@2.0.0
expo-font-interface@2.0.0
expo-google-sign-in@2.0.0
expo-gl@2.0.0
expo-gl-cpp@2.0.0
expo-image-loader-interface@2.0.0
expo-local-authentication@2.0.0
expo-localization@2.0.0
expo-location@2.0.0
expo-media-library@2.0.0
expo-payments-stripe@2.0.0
expo-permissions@2.0.0
expo-permissions-interface@2.0.0
expo-print@2.0.0
expo-react-native-adapter@2.0.0
expo-analytics-segment@2.0.0
expo-sensors@2.0.0
expo-sensors-interface@2.0.0
expo-sms@2.0.0
expo-task-manager@1.0.0
expo-task-manager-interface@1.0.0
expo-asset@2.0.0
expo-module-template@2.0.0
expo@32.0.0

tsapeta committed:
[expokit-npm-package] bump version to v32.0.0

tsapeta committed:
[fastlane] fix `open` function throwing error when trying to open from uri

sjchmiela committed:
[jest-expo] Publish jest-expo@32.0.0

sjchmiela commented on CHANGELOG.md:
```diff
@@ -33,7 +33,7 @@ This is the log of notable changes to the Expo client that are developer-facing.
 - removed obsolete assets from standalone apps by [@sjchmiela](https://github.com/sjchmiela) ([#2850](https://github.com/expo/expo/pull/2850))
 - added support for notifications categories by [@Szymon20000](https://github.com/Szymon20000) and [@sjchmiela](https://github.com/sjchmiela) ([#2316](https://github.com/expo/expo/pull/2316), [#2557](https://github.com/expo/expo/pull/2557))
-- upgraded libraries: `react-native-gesture-handler` to `1.0.12`, `react-native-screens` to `1.0.0-alpha.19`, `react-native-reanimated` to [`72b1792df`](https://github.com/kmagiera/react-native-reanimated/tree/72b1792df7951ed041bdf6d2c422cf03a17346a5) by [@tsapeta](https://github.com/tsapeta) and [@sjchmiela](https://github.com/sjchmiela) ([#2977](https://github.com/expo/expo/pull/2977), [#3078]( ...
+- upgraded libraries: `react-native-gesture-handler` to `1.0.12`, `react-native-screens` to `1.0.0-alpha.19`, `react-native-reanimated` to `1.0.0-alpha.11` (https://github.com/kmagiera/react-native-reanimated/tree/72b1792df7951ed041bdf6d2c422cf03a17346a5) by [@tsapeta](https://github.com/tsapeta) and [@sjchmiela](https://github.com/sjchmiela) ([#2977](https://github.com/expo/expo/pull/2977), [#30 ...
```
```suggestion
- upgraded libraries: `react-native-gesture-handler` to `1.0.12`, `react-native-screens` to `1.0.0-alpha.19`, `react-native-reanimated` to `1.0.0-alpha.11` by [@tsapeta](https://github.com/tsapeta) and [@sjchmiela](https://github.com/sjchmiela) ([#2977](https://github.com/expo/expo/pull/2977), [#3078](https://github.com/expo/expo/pull/3078))
```

sjchmiela committed:
[expokit] Update for final SDK32 (#3107)

* [expokit] Update for final SDK32

* [expokit] Publish expokit@32.0.1-rc.0

* [expokit] Publish expokit@32.0.1

sjchmiela committed:
[wip][docs] Update (#3108)

[docs] add Android ExpoKit upgrade steps and missing docs for notification categories

sjchmiela committed:
Update CHANGELOG.md

Co-Authored-By: tsapeta <tsapeta@users.noreply.github.com>

tsapeta committed:
Merge branch 'master' into sdk-32
